sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/models/ParallelDownloadOptions.java

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.callingserver.models;
+
+import com.azure.communication.callingserver.implementation.Constants.ContentDownloader;
+import com.azure.core.annotation.Fluent;
+
+import java.util.Locale;
+
+/**
+ * This class contains configuration used to parallelize data transfer operations. Note that not all values are used
+ * by every method which accepts this type. Please refer to the javadoc on specific methods for these cases.
+ */
+@Fluent
+public final class ParallelDownloadOptions {
+    private static final String PARAMETER_NOT_IN_RANGE = "The value of the parameter '%s' should be between %s and %s.";
+
+    private Long blockSize;
+    private Long initialBlockSize;
```

Comment:
Do block size and initial block size make sense with regards to downloading? From the Storage perspective these were used during upload operations when chunking the upload into stage block calls with uniform sizes.

Reply:
That's fair, ultimately we want to download everything and reducing the number of variables may reduce complexity.
Thanks.